src/v2/Apps/Search/routes.tsx

```diff
@@ -19,6 +19,7 @@ const prepareVariables = (_params, { location }) => {
 }
 
 const tabsToEntitiesMap = {
+  artist_series: ["ARTIST_SERIES"],
```

Comment:
It would be worth confirming with Brittney/Nicole that we want this to show as a separate tab (vs. under "More")!

Reply:
It should be treated the way collections are (i believe), which has its own tab.